src/main/java/com/hivemq/cli/hivemq/schemas/CreateSchemaTask.java

```diff
@@ -70,13 +75,22 @@ public boolean execute() {
         final Schema schema =
                 new Schema().id(schemaId).type(schemaType).schemaDefinition(definitionBase64).arguments(arguments);
 
+        final Schema createdSchema;
         try {
-            schemasApi.createSchema(schema);
+            createdSchema = schemasApi.createSchema(schema);
         } catch (final ApiException apiException) {
             outputFormatter.printApiException("Failed to create schema", apiException);
             return false;
         }
 
+        if (printVersion) {
+            final JsonObject versionObject = new JsonObject();
+            if (createdSchema.getVersion() != null) {
+                versionObject.add(Schema.SERIALIZED_NAME_VERSION, new JsonPrimitive(createdSchema.getVersion()));
+            }
+            outputFormatter.printJson(versionObject);
```

Comment:
Does it need to be wrapped in JSON?
We could just output the version like 
```
mqtt hivemq create schema ...
1
```
By doing this, the output is directly usable through scripting without having to mingle with the json first.

Reply:
Errors are already printed in JSON, e.g. `{"errors":[{"title":"Schema already present","detail":"The given schema is already present as the latest version for the schema id 'schema-1'."}]}`

I think JSON keeps this consistent, and allows for the possibility of other printed fields in future.

Reply:
Makes sense. Then let's keep it as it is. ✅